tests/Behat/Page/Shop/Page.php

```diff
@@ -21,9 +23,11 @@ public function openOldPath(string $path): void
 
     public function isOnPath(string $path): void
     {
-        $currentPath = parse_url($this->getSession()->getCurrentUrl(), \PHP_URL_PATH);
+        $currentPath = (Uri::createFromString($this->getSession()->getCurrentUrl()))
```

Comment:
Can't you get the path from the Uri object? This seems a bit ugly

Reply:
no, it wont keep the query and the fragment